static-site/src/components/nav-bar/index.tsx

```diff
@@ -98,23 +98,28 @@ const NavBar = ({ minified }: {
   }
 
   const LogoType = () => {
-    const title = "Nextstrain";
-    const rainbowTitle = title.split("").map((letter, i) =>
-      <NavLogoCharacter key={i} $colorIndex={i}>{letter}</NavLogoCharacter>
-    );
-    const SubTitle = styled.div`
-      color: black;
-    `;
-    return (
-      minified ?
-        <div/>
-        :
-        <a href="/">
-          {rainbowTitle}
-          {groupsApp &&
-            <SubTitle>Groups Server</SubTitle>}
-        </a>
-    );
+		const title = "Nextstrain";
+		const rainbowTitle = title.split("").map((letter, i) => (
+			<NavLogoCharacter key={i} $colorIndex={i}>
+				{letter}
+			</NavLogoCharacter>
+		));
```

Comment:
please revert this change, it isn't necessary for the fix and adds unnecessary whitespace.